src/actors/item-specific/weapon.ts

```diff
@@ -82,14 +82,27 @@ export class Weapon extends AbstractEmbeddedItem {
     skillOrigin: string, // Linked skill item origin (uuid)
     actor: RqgActor // The actor that should have the skill
   ): Promise<string> {
-    if (!embeddedSkillId && skillOrigin) {
+    // If the weapon has the aspect (ie one hand, or missile):
+    // embeddedSkillId will be null when dragging from sidebar or directly from compendium
+    // embeddedSkillId will have a value when dragged from one actor to another
+    if (skillOrigin) {
```

Comment:
I think this might give some problems in the case where the skillOrigin doesn't exist anymore? In that case you will get a notification - but there is no way a GM can fix that I think. We can try this out though.